src/index.js

```diff
@@ -928,4 +940,4 @@ export default class Carousel extends Component {
       this.startAutoplay();
     }
   }
-}
+}
```

Comment:
nit: could you add this newline back. Seem like something eslint should have caught and autofixed.